### PR TITLE
Feature1 - Display Flowers

### PR DESCRIPTION
**Pull request by Christykumar**
- [ x ] list all flowers feature added

### CODE REVIEW

File: src/main/java/in/bloomapp/model/Flower.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+	public Flower(String category, String type, int price) {
+		super();
+		this.category = category;
+		this.type = type;
+		this.price = price;
+	}
+
+	@SuppressWarnings("unused")
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/bloomapp/model/Flower.java:29]: Avoid using Private constructor in model class.

File: src/main/java/in/bloomapp/service/FlowerManager.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+import java.util.ArrayList;
+import java.util.List;
+
+import in.bloomapp.model.Flower;
+
+public class FlowerManager {
+	public static final String NATURAL="Natural";
+	public static final String ARTIFICIAL="Artiicial";
+	private static List<Flower> flowers = new ArrayList<>();
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/bloomapp/service/FlowerManager.java:11]: Add Final variable

File: src/main/java/in/bloomapp/service/FlowerManager.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	 * Gets the list of flowers 
+	 * @return
+	 */
+	public static List<Flower> getFlowerList() {
+		
+		return getFlowers();
+	}
+
+
+	public static List<Flower> getFlowers() {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/bloomapp/service/FlowerManager.java:48]: Duplicate method

File: src/test/java/in/bloomapptest/service/DisplayTest.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
+package in.bloomapptest.service;
+
+import static org.junit.Assert.*;
+import java.util.List;
+import org.junit.Test;
+import in.bloomapp.model.Flower;
+import in.bloomapp.service.FlowerManager;
+
+public class DisplayTest {
```
Comment (csys-fresher-batch-2021) [inline src/test/java/in/bloomapptest/service/DisplayTest.java:9]: Rename test class name

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**